src/pages/profile.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Box, useColorModeValue } from '@chakra-ui/react';
+import Sidebar from './components/Sidebar';
+import DashboardHeader from './components/DashboardHeader';
+import UserInfo from './components/UserInformation';
+
+const Profile: React.FC = function Profile() {
+  const { t } = useTranslation('dashboard');
+
+  return (
+    <Box minH="100vh" bg={useColorModeValue('white', 'white')}>
```

Comment:
`useColorModeValue` isn't necessary since the color will always be white

```suggestion
    <Box minH="100vh" bg="white">
```